tests/commetsCounter.test.js

```diff
@@ -0,0 +1,37 @@
+import { fetchShowComments } from '../src/modules/apiComments.js';
+
+const mockFetch = jest.fn(() => Promise.resolve({
+  ok: true,
+  json: () => Promise.resolve([{ comment: 'Example comment' }]),
+}));
+
+global.fetch = mockFetch;
+
+describe('fetchShowComments', () => {
+  afterEach(() => {
+    mockFetch.mockClear();
+  });
+
+  it('should fetch comments for a show successfully', async () => {
+    const showId = 'example-show-id';
+
+    const comments = await fetchShowComments(showId);
+
+    expect(mockFetch).toHaveBeenCalledTimes(1);
+    expect(mockFetch).toHaveBeenCalledWith(
+      `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/RfJpwLbuNZkYmafYdHPm/comments?item_id=${showId}`,
```

Comment:
- [ ]  Since you've already declared a variable for the API earlier, kindly import the variable to avoid code repetition. 👍